fcrepo-persistence-ocfl/src/main/java/org/fcrepo/persistence/ocfl/impl/DefaultOCFLObjectSessionFactory.java

```diff
@@ -55,7 +54,7 @@ private static final File resolveDir(final String systemPropertyKey, final Strin
             return new File(path);
         } else {
             //return default
-            return Paths.get(getProperty(JAVA_IO_TMPDIR), defaultDirectoryName).toFile();
+            return Paths.get("fcrepo-data", defaultDirectoryName).toFile();
```

Comment:
Guessing we'll eventually want a more permanent location for this, probably from config? Maybe its not worthwhile to make a constant yet, hmm.

Reply:
I'll put one in.

Reply:
Actually, I have an update that puts the OCFL storage in the tmp-dir... as intended. 
https://jira.lyrasis.org/browse/FCREPO-3162

The "more permanent location" can currently be defined by the system properties: https://github.com/fcrepo4/fcrepo4/pull/1599/files#diff-f2c25ea789161a21300d5fa03b8a3965R43-R45

Reply:
I have merged FCREPO-3162, so this will need a rebase to pick up the changes.

Reply:
okay - I will not make any changes here.